client/src/inlein/client/tasks/Repl.java

```diff
@@ -0,0 +1,80 @@
+package inlein.client.tasks;
+
+import inlein.client.*;
+import inlein.client.signals.Registerer;
+import java.nio.file.*;
+import java.util.*;
+
+public final class Repl extends Task {
+
+    public static final Repl instance = new Repl();
+
+    private Repl() {
+        super("--repl",
+              "Runs a clojure repl, optionally running a script and/or loading extra dependencies",
+              "Runs a clojure repl, optionally running a script and/or loading extra dependencies");
+    }
+
+    private void die() {
+        System.out.println("repl expects [--deps dep1[;dep2...]] [file [args...]]\n" +
+                           "where deps are semicolon-separated maven coordinates (with group optional if type+classifier also omitted)\n" +
+                           "i.e. [group:]artifact[:type[:classifier]]:version (e.g. clj-time:0.14.0 / org.jclouds:jclouds:1.0:jar:jdk15)");
+        System.exit(1);
+    }
+
+    public void run(ServerConnection conn, String[] args) throws Exception {
+        String coords = "reply:0.3.7";
+        int script = 0;
+        if (args.length > 0 && "--deps".equals(args[0])) {
+            if (args.length < 2) {
+                die();
+            }
+            coords += ";" + args[1];
```

Comment:
Semicolons threw me off here, because I forgot to quote the dependency string. In shells, that will cause some confusing behaviour first time you do it.

I see the rationale for not using commas, but perhaps there's a better symbol out there that may suit us better?

Reply:
Good point, it's not obvious to me what would be a good character to use though. The best I can think of, that's not a special character for bash, is `+`. What do you think of that?